IDE/ECLIPSE/SIFIVE/README.md

```diff
@@ -1,199 +1 @@
-AES-192-CBC-dec     50 KB took 1.260 seconds,   39.677 KB/s
-AES-256-CBC-enc     50 KB took 1.368 seconds,   36.552 KB/s
-AES-256-CBC-dec     50 KB took 1.378 seconds,   36.279 KB/s
-AES-128-GCM-enc     25 KB took 1.225 seconds,   20.412 KB/s
-AES-128-GCM-dec     25 KB took 1.225 seconds,   20.402 KB/s
-AES-192-GCM-enc     25 KB took 1.290 seconds,   19.373 KB/s
-AES-192-GCM-dec     25 KB took 1.291 seconds,   19.366 KB/s
-AES-256-GCM-enc     25 KB took 1.352 seconds,   18.487 KB/s
-AES-256-GCM-dec     25 KB took 1.353 seconds,   18.478 KB/s
-CHACHA               1 MB took 1.006 seconds,    1.020 MB/s
-CHA-POLY           700 KB took 1.032 seconds,  678.045 KB/s
-POLY1305             2 MB took 1.007 seconds,    2.255 MB/s
-SHA                  2 MB took 1.002 seconds,    1.511 MB/s
-SHA-256            525 KB took 1.011 seconds,  519.279 KB/s
-SHA-512            275 KB took 1.017 seconds,  270.477 KB/s
-HMAC-SHA             1 MB took 1.013 seconds,    1.399 MB/s
-HMAC-SHA256        525 KB took 1.019 seconds,  515.020 KB/s
-HMAC-SHA512        275 KB took 1.032 seconds,  266.351 KB/s
-ECC      256 key gen         2 ops took 1.104 sec, avg 551.834 ms, 1.812 ops/sec
-ECDHE    256 agree           2 ops took 1.101 sec, avg 550.400 ms, 1.817 ops/sec
-ECDSA    256 sign            2 ops took 1.173 sec, avg 586.502 ms, 1.705 ops/sec
-ECDSA    256 verify          2 ops took 2.153 sec, avg 1076.294 ms, 0.929 ops/sec
-CURVE  25519 key gen         2 ops took 1.629 sec, avg 814.423 ms, 1.228 ops/sec
-CURVE  25519 agree           2 ops took 1.626 sec, avg 813.156 ms, 1.230 ops/sec
-ED     25519 key gen         1 ops took 1.436 sec, avg 1436.096 ms, 0.696 ops/sec
-ED     25519 sign            2 ops took 2.913 sec, avg 1456.421 ms, 0.687 ops/sec
-ED     25519 verify          2 ops took 5.012 sec, avg 2506.012 ms, 0.399 ops/sec
-Benchmark complete
-```
-
-## Tested Configurations
-- P-RNG (NIST DRBG) with SHA-256
-- SHA 1/256/512
-- AES 128/192/256 CBC/GCM
-- ECC 256 sign/verify/shared secret with fast math or Single Precision (SP) library
-- ED25519/Curve25519
-- HMAC
-- ChaCha20/Poly1305
-
-## Known Caveats
-- If you find the wolfcrypt test stuck on early_trap_vector error, it is like related to memory issues
-- Using the `__stack_size` default value of 0x400 will not be enough for the ECC test to pass.
-The `IDE/ECLIPSE/SIFIVE/Makefile` overwrites the value with 0x1000 (4 KBytes)
-- Enabling RSA will cause the ECC test to fail due to memory shortage.
-
-## References
-
-The test results were collected from a SiFive reference platform target with the following hardware, software and tool chains:
-- HiFive1 Rev A/Rev B: HiFive1 Development Board with the Freedom Everywhere SoC, E300
-- freedom-e-sdk
-- wolfssl [latest version](https://github.com/wolfSSL/wolfssl)
-
-For more information or questions, please email [support@wolfssl.com](mailto:support@wolfssl.com)
+This folder has moved to `IDE/RISCV/SIFIVE-HIFIVE1`.
```

Comment:
If the next release contains a description of the folder move, why not remove IDE/ECLIPSE/SIFIVE/* all together now?

Reply:
We may already have links to this page... I want to make sure folks know where it went.

The include.am files are correct because I already added `IDE/RISCV/include.am` previously.

Reply:
OK. Thanks. Looks good.